ops/framework.py

```diff
@@ -220,19 +220,19 @@ def defer(self) -> None:
 
         The general desire to call ``defer()`` happens when some precondition
         isn't yet met. However, care should be exercised as to whether it is
-        better to defer this event so that you see it again, or whether it is
+        better to defer this event so that it is seen again, or whether it is
         better to just wait for the event that indicates the precondition has
         been met.
 
-        For example, if ``config-changed`` is fired, and you are waiting for
-        different config, there is no reason to defer the event because there
-        will be a *different* ``config-changed`` event when the config actually
-        changes, rather than checking to see if maybe config has changed prior
-        to every other event that occurs.
+        For example, if ``config-changed`` is fired, and handling the event
+        cannot complete without a different config, there is no reason to defer
+        the event because there will be a *different* ``config-changed`` event
+        when the config actually changes, rather than checking to see if maybe
+        config has changed prior to every other event that occurs.
```

Comment:
This is a fine change, though I think this paragraph needs a rewrite. Some of the phrasing is odd/confusing. Feel free to have a shot if you want, or we can leave it for now.

Reply:
```suggestion
        For example, if handling a config change requires that two configs are
        changed, there's no reason to defer the first ``config-changed``
        because there will be a *second* ``config-changed`` event fired when
        the other config changes. Instead, check whether the config has changed
        prior to other events where the config is relevant.
```

Is this better? The last phrase in the original isn't super clear to me, so I'm not sure if the last sentence in the above is correct.

Reply:
Yeah, that's a lot better. I think we can actually just leave the last sentence off.

One nit: let's say "config values" rather than "configs". So "two configs" -> "two config values" and "other config changes" -> "other config value changes".